.github/workflows/e2e-android-detox.yml

```diff
@@ -0,0 +1,80 @@
+name: Detox E2E Android Tests PR
+
+on:
+    pull_request:
+      branches:
+        - main
+      types:
+        - labeled
+
+concurrency:
+    group: "${{ github.workflow }}-${{ github.event.pull_request.number }}-${{ github.event.label.name }}"
+    cancel-in-progress: true
+
+jobs:
+    build-android-apk:
+        if: github.event.label.name == 'E2E Android tests for PR'
+        runs-on: ubuntu-latest-8-cores
+        env:
+            ORG_GRADLE_PROJECT_jvmargs: -Xmx8g
+        steps:
+            - name: Prune Docker
```

Comment:
Is this because of disk space being full?

Perhaps we should add a comment on this as well.

Reply:
Yes,
In CI, I'm facing Out of memory , when the buld command runs.

I'll add a comment.